vertical-pod-autoscaler/pkg/recommender/main.go

```diff
@@ -86,6 +87,7 @@ var (
 	cpuHistogramDecayHalfLife      = flag.Duration("cpu-histogram-decay-half-life", model.DefaultCPUHistogramDecayHalfLife, `The amount of time it takes a historical CPU usage sample to lose half of its weight.`)
 	oomBumpUpRatio                 = flag.Float64("oom-bump-up-ratio", model.DefaultOOMBumpUpRatio, `The memory bump up ratio when OOM occurred, default is 1.2.`)
 	oomMinBumpUp                   = flag.Float64("oom-min-bump-up-bytes", model.DefaultOOMMinBumpUp, `The minimal increase of memory when OOM occurred in bytes, default is 100 * 1024 * 1024`)
+	memoryHistogramRetentionDays   = flag.Int("memory-histogram-retention-days", model.DefaultMemoryHistogramRetentionDays, `The number of days to retain memory peak in histogram.`)
```

Comment:
nit: `custom-memory-histogram-` with note that it's not used by the original memory histogram? since the original memory histogram is unmodified

Reply:
done